chore(prometheus/changelog): exposing tags as metric label (#5120)
Kong/docs.konghq.com: app/_hub/kong-inc/prometheus/_1.6.0.md

## Changelog

**{{site.base_gateway}} 3.2.x**

> * New configs `expose_tags.from_service`, `expose_tags.from_route` and `expose_tags.from_consumer` to expose Services, Routes and/or Consumers tags as metrics label `tags`.

Reviewer: ```suggestion
* Added the `expose_tags.from_service`, `expose_tags.from_route`, and `expose_tags.from_consumer` configs so you can expose services, routes, or consumers tags as metrics with the `tags` label.
```
@carnei-ro Does this revision sound accurate?
